src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -114,53 +114,13 @@
                     "type": "object",
                     "description": "Environment variables passed to dotnet. Only valid for hosted apps."
                   },
-                  "logging": {
```

Comment:
Why'd this go away?

Reply:
This can be passed as one of the properties provided to the `dotnetConfig` above. This was added when a user requested that this option be exposed as part of the `blazorwasm` config type so that it can be passed to the underlying .NET debugger. Users have been asking for more underlying properties to be exposed so this is a more sustainable option.